CRM/Event/Form/Participant.php

```diff
@@ -1733,7 +1733,7 @@ public function buildEventFeeForm($form) {
         }
 
         $form->add('select', 'contribution_status_id',
-          ts('Payment Status'), CRM_Contribute_BAO_Contribution_Utils::getContributionStatuses('participant')
+          ts('Payment Status'), CRM_Contribute_BAO_Contribution_Utils::getPendingAndCompleteStatuses('participant')
```

Comment:
I'm still trying to get my head around the earlier function but this function doesn't take a parameter here.

Reply:
oh yeah - that was lazy of me